include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
Ok, I'll cave...but I think my point still stands that the separation is actually less as three classes are now involved for one feature.

In hopes to of saving me time later are there any complaints to roll in my suggestions for PR #85 while I'm at it?